### PR TITLE
Adding support for multiple css and js files

### PR DESCRIPTION
**Pull request by emcniece**
Threw in a couple of functions for handling arrays of css and js files:

```js
var html = createHTML({
  css: ['sheet1.css', 'sheet2.css'],
  script: ['script1.css', 'script2.css'],
})
```

Includes tests, linting and a README update.

I didn't update package.json `readme` though... not sure how that is generated.

Resolves https://github.com/sethvincent/create-html/issues/12

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,13 +1,43 @@
...
+function buildScripts (scripts, async) {
+  var output = ''
+  if (!scripts) return output
+
+  if (typeof scripts === 'string') {
+    scripts = [scripts]
+  }
+
+  scripts.forEach(function (script) {
+    output += !async
```
Comment (sethvincent) [inline index.js:27]: What do you think about adding a new line between each `<script>` and `<link>` tags?

That would make the end result a little easier to look at if a user has to go back and make changes later (I'm thinking of the cli use case here).

File: README.md [hunk 2]
```diff
@@ -113,6 +113,18 @@ var stream = fromString(html)
...
 
+
+### Multiple CSS and Javascript Files
+
+Multiple script and stylesheets can be added by sending an array instead of a string:
+
+```js
+var html = createHTML({
+  css: ['sheet1.css', 'sheet2.css'],
+  script: ['script1.css', 'script2.css'],
```
Comment (sethvincent) [inline README.md]: Could you do a quick update here to change the `.css` instances to `.js` and remove the trailing comma?

**Review by sethvincent — CHANGES_REQUESTED** [review]
Thanks this will be useful! Just a couple of small changes and we can merge this.

**Comment by emcniece** [comment]
Changes made, thanks for the review!

The generated HTML has some extra whitespace around the link/script tags now as a result of the newlines. This doesn't seem too bad, and keeping these spaces avoids extra logic for excluding newlines on the final array elements during iteration. That's not impossible to implement, but it doesn't seem crucial.

**Review by sethvincent — APPROVED** [review]

**Comment by sethvincent** [comment]
Cool thanks!

About the extra whitespace: not a big deal. I've been thinking about stripping all the empty lines before returning the string, which would fix up this issue and deal with the empty lines that result from empty arguments.

**Comment by sethvincent** [comment]
📦 4.1.0

**Pull request merged**